### DISCUSSION TITLE
How to properly delete lots of small file?

### DISCUSSION THREAD

**Discussion by ibigbug** [minio/minio#18145] [Q&A]
I have a lot of small old files and I want to delete them via
<img width="341" alt="image" src="https://github.com/minio/minio/assets/543405/4d6a98a4-3ce0-4f17-9c5b-d2afb1b9535d">

```
 mc rm --older-than 90d --force --bypass --recursive FOLDER
```

but it stuck for ever and never succeeds

update: the total size is about 200G

**Comment by klauspost** [comment]
`mc rm` deletes individual objects and will take some time depending on your system.

`mc rb --force` tends to be a lot faster.

**Reply by ibigbug** [reply]
it's doing different stuff though
